frontend/src/index.css

```
@@ -203,6 +203,43 @@ body {
+  margin-bottom: 0.85rem;
+}
+
+.draft-recovery-banner__discard-link {
+  background: none;
+  border: none;
+  color: #166534;
+  text-decoration: underline;
+  cursor: pointer;
+  font-weight: 600;
+  padding: 0;
+  font: inherit;
```

Stylelint (17.12.0)

[error] Overridden property "font-weight" by shorthand "font" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)